.github/workflows/run-validation.yml

```diff
@@ -0,0 +1,67 @@
+name: Run validation with RoboValidate
+
+on:
+    # Run on any branch so validate branch can always run.
```

Comment:
This is just one example, but 'Contact Us' has a body field that has markdown in it. This text MUST be converted to HTML by using this callback first http://digitalgov.lndo.site/admin/convert-text. It should turn the MD to HTML.
You'd use the second option here 
![image](https://github.com/user-attachments/assets/bedf2fe5-d5a5-4d7e-a922-a40acf15fa9f)